src/cdk-experimental/listbox/listbox.ts

```diff
@@ -123,18 +160,40 @@ export class CdkOption implements ListKeyManagerOption, Highlightable {
       'role': 'listbox',
       '(keydown)': '_keydown($event)',
       '[attr.aria-disabled]': '_disabled',
+      '[attr.aria-multiselectable]': '_multiple',
+      '[attr.aria-activedescendant]': '_getAriaActiveDescendant()'
     }
 })
-export class CdkListbox implements AfterContentInit, OnDestroy {
+export class CdkListbox implements AfterContentInit, OnDestroy, OnInit {
 
   _listKeyManager: ActiveDescendantKeyManager<CdkOption>;
+  _selectionModel: SelectionModel<CdkOption>;
+
   private _disabled: boolean = false;
+  private _multiple: boolean = false;
+  private _useActiveDescendant: boolean = true;
+  private _activeOption: CdkOption;
+  private readonly _destroy = new Subject<void>();
```

Comment:
I think this is supposed to be a ReplaySubject(1)?

Reply:
I'm not sure what the difference is. I'll look into it.